demo/src/pages/demos/3dcarousel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from "react";
+
+import Axes, { PanInput, WheelInput } from "../../../../src/index";
+import "../../css/demos/3dcarousel.css";
+
+function Carousel() {
+  useEffect(() => {
+    var carousel = document.getElementById("carousel");
+    var COUNT = 8;
+    var ANGLE = 360 / COUNT;
+
+    // 1. Initialize eg.Axes
+    var axes = new Axes({
+      rotate: {
+        range: [0, 360],
+        circular: true
+      }
+    });
+
+    // 2. attach event handler
+    axes.on({
+      "change": function(e) {
+        var pos = e.pos;
+
+        carousel.style[Axes.TRANSFORM] = "translateZ(-253px) rotateY(" + (pos.rotate) + "deg)";
+      },
+      "release": function (e) {
+        var destPos = e.destPos;
+
+        e.setTo({"rotate": Math.round(destPos.rotate/ANGLE) * ANGLE }, 500);
+      }
+    });
+
+    // 3. Initialize a inputType and connect it
+    axes.connect("rotate", new PanInput("#carouselWrapper")).connect("rotate", new WheelInput("#carouselWrapper", {useNormalized: false}));
+  });
+
+  return (
+    <div className="demobox" codepen="3dcarousel">
+      <p>You can create a carousel by controlling the animation.</p>
+      <div id="carouselWrapper">
+        <div className="list_container">
+          <div id="carousel" style={{ transform: `translateZ(-253px)` }}>
+            <figure style={{ transform: `rotateY(0deg) translateZ(253px)` }}>
+              <div id="list_cd" style={{ backgroundImage: "url("+require("@site/static/img/demos/3dcarousel/music1-min.jpg").default+")" }}>
```

Comment:
Using backtick (\`) looks wrong here and there.
transform's value should be covered with \", and backgroundImage should rather use backtick.